insta-unfollower.py

```diff
@@ -219,7 +223,11 @@ def main():
 
         time.sleep(random.randint(2, 4))
 
-    connected_user = get_user_profile(os.environ.get('INSTA_USERNAME'))
+    if len(sys.argv) > 1:
+        connected_user = sys.argv[1]
+    else: 
+        get_user_profile(os.environ.get('INSTA_USERNAME'))
```

Comment:
Create a variable `username` equal to `sys.argv[1`] or `os.environ.get('INSTA_USERNAME')` depending of arguments/environment and call  `connected_user = get_user_profile(username)`